python/cugraph/cugraph/structure/graph_classes.py

```diff
@@ -683,6 +683,27 @@ def add_nodes_from(self, nodes):
         """
         self._Impl._nodes["all_nodes"] = cudf.Series(nodes)
 
+    def density(self):
+        """
+        Return the density of the graph.
+        Density is the measure of how many edges are in the graph versus
+        the max number of edges that could be present.
+        Note:  This function does not work on multigraph since there
+        is no theoretical max number of edges
+        """
```

Comment:
This docstring will need to follow the format we use for public APIs, which includes an example, so it can be properly rendered by the API docs generating tools.  [Here's one](https://github.com/rapidsai/cugraph/blob/a529d762e18728223ac0f2f5c74eb3974ddcd7c0/python/cugraph/cugraph/structure/graph_classes.py#L618-L636) for reference.

Reply:
fixed